image/tarexport/load.go

```diff
@@ -201,15 +242,40 @@ func (l *tarexporter) loadLayer(filename string, rootFS image.RootFS, id string,
 	}
 
 	inflatedLayerData, err := archive.DecompressStream(r)
+
+	layerData := inflatedLayerData
+	if deltaBase != nil {
+		pR, pW := io.Pipe()
+		go func() {
+			tr := tar.NewReader(inflatedLayerData)
```

Comment:
To answer my above question, I guess `archive` was already imported from elsewhere, and you needed to add tar for this line.

Reply:
Right, exactly this, @klutchell !